packages/ses/test/test-v8-callsite-properties.js

```diff
@@ -8,10 +8,14 @@ test('callSite properties', t => {
     let sst;
     const orig = Error.prepareStackTrace;
     try {
-      const pst = (_err, sst0) => sst0;
+      const pst = (_err, sst0) => {
+        sst = sst0;
+        return '';
+      };
       Error.prepareStackTrace = pst;
       const e = Error();
-      sst = e.stack;
+      // eslint-disable-next-line no-void
+      void e.stack;
```

Comment:
Please help me understand this change and how it relates to this PR.

Reply:
This test was abusing the `depd` hack of `prepareStack` returning the sst directly. I didn't want to perpetrate it, so I refactored the test to instead capture the sst in another way.

Reply:
Oh! Resolving.